apis/stardust/types/index.d.ts

```diff
@@ -41,6 +41,17 @@ export function useEffect(effect: stardust.EffectCallback, deps?: any[]): void;
  */
 export function useMemo<T>(factory: ()=>T, deps: any[]): T;
 
+/**
+ * Creates a reference to a value not needed for rendering
+ * 
+ * While Nebula does not have a virtual DOM, it is still useful
+ * to have a reference to an object that is retained across
+ * renders and in it self does not trigger a render.
+ * @param initialValue The initial value.
+ */
+export function useRef<R>(initialValue: R): {
+};

```

Comment:
return type is still not correct
(maybe could be fixed by adding and using a generic Ref type, similar to SetStateFn used by useState?)

Reply:
tricky to get right, used an interface now